.github/scripts/action_entrypoint.sh

```
@@ -3,44 +3,93 @@ set -e
+    # Extract verification result
+    if echo "$OUTPUT" | grep -q "✅ VERIFIED"; then
+        echo "verified=true" >> $GITHUB_OUTPUT
+    elif echo "$OUTPUT" | grep -q "❌"; then
+        echo "verified=false" >> $GITHUB_OUTPUT
+    else
+        echo "verified=false" >> $GITHUB_OUTPUT
+    fi
 else
-    echo "verified=false" >> $GITHUB_OUTPUT
+    # Rely on process exit code for other tools
+    if [ $? -eq 0 ]; then
```

Shellcheck (0.11.0)

[warning] This $? refers to a condition, not a command. Assign to a variable to avoid it being overwritten.

(SC2319)